src/main/java/com/github/manolo8/darkbot/BackpageManager.java

```diff
@@ -43,13 +47,22 @@ public void run() {
                 waitTime = validTick();
             }
 
+            if(checkDrones) {
+                hangarManager.checkDrones();
+                checkDrones = false;
+            }
+
             if (sidStatus == 302) break;
 
             this.waitTime = (int) (waitTime + waitTime * Math.random());
             sleep(waitTime);
```

Comment:
You're gonna have to change this.
This was fine when SID checking was all that was done, but it's no longer valid now. I would sleep for 10 seconds at the time, and just have the if checkDrones, alongside a if waitTime condition to do the "validTick()", wich tbh, validTick does not fit it well at all anymore